### DISCUSSION TITLE
MDMA + Alcohol

### DISCUSSION THREAD

**Discussion by zelixir25** [TripSit/drugs#76] [Drug Interactions]
The combination of MDMA and alcohol may increase strain on the cardiovascular system and lead to increased alcohol consumption. MDMA may also impair one's ability to recognize their level of drunkenness, leading to increased alcohol consumption and poor decision-making. Alcohol may also increase the risk of dehydration and hyperthermia (increased body temperature) when combined with MDMA. 

Hamida, S. B., Plute, E., Cosquer, B., Kelche, C., Jones, B. C., & Cassel, J.-C. (2008). Interactions between ethanol and cocaine, amphetamine, or MDMA in the rat: thermoregulatory and locomotor effects. Psychopharmacology, 197(1), 67–82. https://doi.org/10.1007/s00213-007-1007-5

Hernández-López, C., Farré, M., Roset, P. N., Menoyo, E., Pizarro, N., Ortuño, J., Torrens, M., Camı́, J., & Torre, R. de la. (2002). 3,4-Methylenedioxymethamphetamine (Ecstasy) and Alcohol Interactions in Humans: Psychomotor Performance, Subjective Effects, and Pharmacokinetics. Journal of Pharmacology and Experimental Therapeutics, 300(1), 236–244. https://doi.org/10.1124/jpet.300.1.236

Ramaekers, J. G., & Kuypers, K. P. C. (2006). Acute Effects of 3,4-Methylenedioxymethamphetamine (MDMA) on Behavioral Measures of Impulsivity: Alone and in Combination with Alcohol. Neuropsychopharmacology, 31(5), 1048–1055. https://doi.org/10.1038/sj.npp.1300894

van Amsterdam, J., Brunt, T. M., Pierce, M., & van den Brink, W. (2021). Hard Boiled: Alcohol Use as a Risk Factor for MDMA-Induced Hyperthermia: a Systematic Review. Neurotoxicity Research, 39(6), 2120–2133. https://doi.org/10.1007/s12640-021-00416-z

**Comment by utaninja** [comment]
These sources look good to me. Given how often the combination is used, what are your thoughts on adding part of the current combination message which says "Approach this combination with caution, moderation and sufficient hydration."

**Reply by zelixir25** [reply]
I'm looking now at the current phrasing on the /combo command which is:

> These combinations are not usually physically harmful, but may produce undesirable effects, such as physical discomfort or overstimulation. Extreme use may cause physical health issues. Synergistic effects may be unpredictable. Care should be taken when choosing to use this combination.

The combo.tripsit.me text says:

> Both MDMA and alcohol cause dehydration. Approach this combination with caution, moderation and sufficient hydration. More than a small amount of alcohol will dull the euphoria of MDMA

Both of these hit on important aspects of the combination. I think something like:
> MDMA can inhibit alcohol's intoxicating effects leading to increased drinking, which may put individuals at risk for adverse physical effects and poorer decision making. Alcohol may also reduce some of the euphoric effects of MDMA. This combination also carries an increased risk of hyperthermia and dehydration and places increased strain on the cardiovascular system. Approach this combination with caution, moderation, and sufficient hydration.

**Reply by utaninja** [reply]
Looks good to me. Pushing

**Comment by utaninja** [comment]
Pushed to https://github.com/TripSit/drugs/pull/130
